package-lock.json

```diff
@@ -1,5060 +0,0 @@
-{
```

Comment:
in your local project, can you run `git checkout package-lock.json` to bring it back. The main repo has one so we should keep it probably. Unless you removed it on purpose.